suprb/optimizer/rule/ns/ns.py

```diff
@@ -30,12 +29,15 @@ class NoveltySearch(RuleGeneration):
             The amount of offspring from each population that get selected for the next generation.
         lmbda: int
             Each generation lambda children will be generated.
+        roh: int
+            Each generation roh rules will be added to the archive (depending on archive type)
```

Comment:
Can you, based on the papers, provide some defaults here?

Reply:
In the paper a range between 1 and 199 is used (they use a fixed population size of 200). Best results have been ~15, but this is probably dependent on the population. I also put it to 15, but this parameter should definitely be tuned to see what yields the best results

Reply:
then I would add `For population sizes of 200, a value of 15 is generally recommended. However, this value is highly sensitive to a multitude of factors (and novelty search mechanisms, such as the specific archiving technique)`